src/algorithm/butterflies.rs

```diff
@@ -696,23 +760,10 @@ mod unit_tests {
 	use super::*;
 	use test_utils::{random_signal, compare_vectors, check_fft_algorithm};
 	use algorithm::DFT;
-	use num::Zero;
-
-	#[test]
-	fn test_butterfly2() {
```

Comment:
Before, it was different than all the other butterfly instances because its constructor didn't take the `inverse` parameter. So it had to be separate. But, because of the addition of the IsInverse trait, Butterfly2 needs to know whether it was created as an inverse or not, meaning it now takes inverse in the constructor.

So now, it's identical to all the others, and can be tested via the macro. If you check line 783 of the additions, you can see it tested there.